packages/synapse-react-client/src/utils/functions/SqlFunctions.ts

```diff
@@ -29,6 +30,10 @@ export const getIgnoredQueryFilterSearchParamKey = (
 ) => {
   return `__${namespace ?? ''}_${key}`
 }
+
+// Special search parameter key that will automatically apply a FTS search term to a Query Wrapper if present
+export const FTS_SEARCH_TERM = 'FTS_SEARCH_TERM'
```

Comment:
Here's the other magic.  When a special search parameter is present, add the TextMatchesQueryFilter filter to the query additionalFilters

Reply:
I do want to refactor this at some point so the search params are translated to filters _before_ passing them to QueryWrapper components as props. My reasoning is that the searchParam -> filter translation is kind of unintuitive and is more related to the app/routing logic than the component logic.

I don't think anything you changed here makes that more complicated, I guess I just want to know if you have an opinion about that change while we're both looking at this :slightly_smiling_face:

Reply:
I absolutely support the proposed change